### PR TITLE
Create placeholder package for the `@cerner/terra-cli` package

### PR DESCRIPTION
**Pull request by mjhenkes**
### Summary

This is the placeholder for Terra cli post terra-toolkit-boneyard. This cli will provide a common entry point to the terra commands for package management as well as for testing.

### Additional Details

The packages is mostly blank and was copied from the eslint config terra package. Content and dependencies will be filled in later. This package is ignored by lerna until we're ready for a release.

@cerner/terra

[CONTRIBUTORS.md]: ../blob/main/CONTRIBUTORS.md
[License]: ../blob/main/LICENSE

### CODE REVIEW

File: packages/terra-cli/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@cerner/terra-cli",
+  "version": "4.1.0",
```
Comment (StephenEsser) [inline packages/terra-cli/package.json]: Starting big

Comment (mjhenkes) [reply packages/terra-cli/package.json]: oops 😄 
```suggestion
  "version": "0.0.0",
```

File: packages/terra-cli/package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    "Cerner",
+    "Terra"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-toolkit/issues"
+  },
+  "engines": {
+    "node": "^10.13.0 || >=11.10.1"
```
Comment (StephenEsser) [inline packages/terra-cli/package.json:26]: Do you remember why we start the upper limit at a (seemingly) random version of 11.x.x?

**Review by StephenEsser — APPROVED** [review]

File: packages/terra-cli/package.json (hunk 2 above)
Comment (mjhenkes) [reply packages/terra-cli/package.json:26]: probably because of eslint, I imagine we could drop this back in this package.

**Review by yuderekyu — APPROVED** [review]

**Review by ryanthemanuel — APPROVED** [review]

**Review by benbcai — APPROVED** [review]

**Review by tbiethman — APPROVED** [review]

**Pull request merged**